src/common/snippets/tests/src/lowered/pass/extracted_loop_invariants.cpp

```diff
@@ -263,14 +263,14 @@ TEST_F(ExtractLoopInvariantsTest, ExtractedLoopInvariantsFromInnermostToLoopOuts
         auto add = linear_ir->push_node<ov::opset10::Add>(param_0.second, broadcastmove.second);
         init_expr_descriptors(*add.first, {subtensor, subtensor, subtensor}, {layout, layout, layout});
         auto result = linear_ir->push_node<ov::opset10::Result>(add.second);
-        create_and_add_unified_loop_info(linear_ir, broadcastmove.first, result.first, 3, 1,
-                                        {LoopPort((*broadcastmove.first)->get_input_port(0), true, 1),
-                                         LoopPort((*add.first)->get_input_port(0), true, 1)},
-                                        {LoopPort((*add.first)->get_output_port(0), true, 1)});
         create_and_add_unified_loop_info(linear_ir, broadcastmove.first, result.first, 512, vector_size,
                                         {LoopPort((*broadcastmove.first)->get_input_port(0), true, 0),
                                          LoopPort((*add.first)->get_input_port(0), true, 0)},
                                         {LoopPort((*add.first)->get_output_port(0), true, 0)});
+        create_and_add_unified_loop_info(linear_ir, broadcastmove.first, result.first, 3, 1,
```

Comment:
Changed the order because the loops with inner `dim_idx` should be innermost loops.
For more details, please see the ticket 112196